### ISSUE TITLE
Trying to write a memory viewer window

### ISSUE THREAD

**Issue by DanB91** [ocornut/imgui#920]
Hi all,

I am currently trying to implement a memory viewer similar to the Memory editor here: https://github.com/ocornut/imgui/wiki/memory_editor_example

The only difference is I only need to view and not edit.  Also I don't need the  customizable rows DragInt at bottom left hand corner.

I have got the viewer working, but currently 2 (minor) issues I can't seem to figure out.  Currently I have another window up along with the memory viewer.  The program currently looks like this:

<img width="637" alt="screen shot 2016-11-26 at 2 26 08 am" src="https://cloud.githubusercontent.com/assets/1284373/20638828/c22c49fc-b37f-11e6-9498-29a0d7248a1f.png">

As you can see the Memory viewer is overlapping the first window on start up.  I move the memory viewer to look at both windows side-by-side, but I'd like this done automatically on start up.  

I've tried doing the following:

```
  //....Previous window defined here.....
  ImVec2 debugDataSize = ImGui::GetWindowSize();
  ImGui::End();
        
  if (ImGui::Begin("Memory Viewer", nullptr)) {
    ImGui::SetWindowPos(ImVec2(debugDataSize.x, 0), ImGuiSetCond_Once);
```

But this doesn't seem to move the window at all.  Only if use ImGuiSetCond_Always, does this work, but then I can't move the Memory viewer window.  I am sure there is something obvious I am missing.

Thanks!

**Comment by ocornut** [comment]
Unfortunately the size of a window isn't known on the first frame it is submitted (as it needs initial content to compute a starting size). So you either need to wait one more frame either compute the size/position yourself. Note that using the .ini files the position are saved anyway so it may not matter to you so much to get initial positions right.

**Comment by DanB91** [comment]
Makes sense thank you!    Don't see it in Issues, but has there been an enhancement request made for a "ImGuiSetCond_AfterFirstFrame"?

**Comment by ocornut** [comment]
I don't think I would want to add a ``ImGuiSetCond_AfterFirstFrame` as it'd be a bit of an unobvious workaround against the current behavior and you could easily replicate it yourself if you really want the initial positioning.
